app/services/live.js

```diff
@@ -70,11 +70,26 @@ export default class LiveService extends Service {
 
   onConnection(isConnected) {
     this.isJoined = isConnected;
+    if (this.isJoined) {
+      window.addEventListener('beforeunload', this.onCloseAlert);
+    } else {
+      window.removeEventListener('beforeunload', this.onCloseAlert);
+    }
     if (isConnected) {
       this.isLoading = false;
     }
   }
 
+  onCloseAlert(event) {
+    console.log('Before Unload Starts');
```

Comment:
remove console logs

Reply:
removed the logs.

Reply:
You've removed comments instead of logs

Reply:
I have removed all dead code now (comments and logs)